zuul.d/edpm_periodic.yaml

```diff
@@ -11,6 +11,18 @@
     parent: cifmw-crc-podified-edpm-baremetal
     vars: *edpm_vars
 
+- job:
+    name: periodic-edpm-build-push-images-centos-9-master
+    parent: cifmw-base-edpm-build-images
+    vars:
+      buildah_login: true
```

Comment:
I think it is unused variable.

Reply:
There is when condition in a pre based on this var: https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/playbooks/crc/podman.yaml#64

There is another task above for podman login, maybe the podman login is enough.

I can try removal.

Reply:
trying var removal, will revert if job fails

Reply:
Removed this var and we were still able to push container images, so podman login was enough[1]

[1] https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/playbooks/crc/podman.yaml#52